iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/scalable/json/JsonScalingParametersTest.java

```diff
@@ -40,6 +42,7 @@ void testDeserializerV1dot1() {
         assertEquals(ONESHOT, parameters.getPriority());
         assertTrue(parameters.isReconnect());
         assertFalse(parameters.isAllowsGeneratorOutOfActivePowerLimits());
+        assertEquals(2, parameters.getIgnoredInjectionIds().size());
```

Comment:
This test shouldn't be altered since it checks that version 1.0 files can still be read.

Can you also:
- change this method's name to reflect its content (e.g. `testDeserializerV1dot0`)
- create a method to test the import of v1.1 files (since `roundTrip()` will test it for the current – v1.2 – version)?

Reply:
Ended up renaming and creating tests to have one test for 1.0, one for 1.1, one for 1.2, and keep one round trip test for the latest version (so currently 1.2).